docs/aerodromes/Alice Springs (YBAS)/coordination.md

```diff
@@ -38,4 +38,10 @@ ML-ASP_CTR will coordinate all arrivals into Alice Springs Prior to handing over
 
 !!! example
     **ML-ASP_CTR** -> **AS_TWR**: "JST734, A320, estimates Alice Springs 52, 189 radial, 5000ft, number 2”
-    **AS_TWR** -> **ML-ASP_CTR**: "JST734"
\ No newline at end of file
+    **AS_TWR** -> **ML-ASP_CTR**: "JST734"
+
+## Standard Assignable Levels
+
+Aircraft departing from Alice Springs shall be assigned `A060` or `RFL` if lower.
```

Comment:
I have a feeling it might be 4000 for departure and 5000ft for arrivals.
unless you have different documentation that says otherwise

Reply:
should be A080 for arrivals, and lower of A070 or RFL for departures